code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -306,6 +306,11 @@
 /obj/spacepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
 	to_chat(S, "<span class='warning'>Destroying this vehicle would destroy us. Aborting.</span>")
 
+/obj/machinery/clonepod/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+	if(occupant)
+		to_chat(S, "<span class='warning'>Destroying this machine while it is occupied would result in biological and sentient resources to be harmed. Aborting.</span>")
+		return ..()
```

Comment:
you don't want to return ..()

You want to call ..() if there isn't a return

What you have means that the only way a swarmer can deconstruct a clonepod is if it does have an occupant in it.

Reply:
Oh I get it. I'm dumb